Formula/mad.rb

```diff
@@ -22,12 +22,254 @@ class Mad < Formula
     sha256 "7bd46d4da0f695b3a5bcc899b7139f14d11741f2e47d34f21a984f9bab953c81" => :mavericks
   end
 
+  depends_on "autoconf" => :build
+  depends_on "automake" => :build
+  depends_on "libtool" => :build
+
+  resource "sfx.mp3" do
+    url "https://raw.githubusercontent.com/chromium/chromium/master/media/test/data/sfx.mp3"
+    sha256 "46364ebce1b9b2f278b35a7dc05b10427a33c073cf085cfa1a4ab33f3d307565"
+  end
+
   def install
+    touch "NEWS", "AUTHORS", "ChangeLog"
+    system "autoreconf", "-fiv"
     system "./configure", "--disable-debugging", "--enable-fpm=64bit", "--prefix=#{prefix}"
     system "make", "CFLAGS=#{ENV.cflags}", "LDFLAGS=#{ENV.ldflags}", "install"
     (lib+"pkgconfig/mad.pc").write pc_file
   end
 
+  test do
+    testpath.install resource("sfx.mp3")
+    (testpath/"minimad.c").write <<~EOS
```

Comment:
Can you download the file as a resource too?

Reply:
It is from `libmad` tarball but I cannot find an official place to download it as single file. Is there a special way to refer to the source or should I just re-download the tarball and extract from there?

Reply:
We can copy it in the `install` method, like this:
`pkgshare.install "examples/wherever/minimad.c"`

Reply:
Done. I also fixed the touch error and switched to using test.mp3 in test_fixtures to reduce complexity of the test.